composer.json

```diff
@@ -35,7 +35,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.0.x-dev"
```

Comment:
Including the minor version is better, otherwise compose will resolve things like 2.55.x to master, when in fact master is 2.0.x series.

Reply:
i was afraid we will forget to up this to 2.1.x-dev once we start that version. but will revert this change then.